docs/tutorials/FlowTutorial.md

```diff
@@ -0,0 +1,1599 @@
+    - [Timing Optimization Using repair_design](#Timing-Optimization-Using-repair_design)
+    - [Timing Optimization Using repair_timing](#Timing-Optimization-Using-repair_timing)
+    - [Timing Optimization Based On Multiple Corners](#Timing-Optimization-Based-On-Multiple-Corners)
+    - [Fixing Setup Violations](#Fixing-Setup-Violations)
+    - [Fixing Hold Violations](#Fixing-Hold-Violations)
+  - [Clock Tree Synthesis](#Clock-Tree-Synthesis)
+    - [Reporting Clock Skews](#Reporting-Clock-Skews)
+    - [Reporting CTS Metrics](#Reporting-CTS-Metrics)
+
+## Introduction
+
+This document describes a tutorial to run the complete
+OpenROAD based flow from RTL-to-GDS using [OpenROAD Flow
+Scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts).
+This tutorial also includes examples of useful design explorations and
+manual usage in key flow stages to help users gain a good understanding of
+the [OpenROAD](https://openroad.readthedocs.io/en/latest/main/README.html)
+application flow, data organization, GUI and commands.
+
+This is intended for:
+
+-   Beginners or new users with some understanding of basic VLSI
+    design flow. Users will learn the basics of installation to use
+    [OpenROAD-flow-scripts](https://openroad.readthedocs.io/en/latest/user/GettingStarted.html)
+    for the complete RTL-to-GDS flow.
+-   Users already familiar with the OpenROAD application and flow but would
+    like to learn more about specific features and commands.
+
+## User Guide
+
+-   This tutorial requires a specific directory structure built by
+    OpenROAD-flow-scripts (ORFS). Do not modify this structure or
+    underlying files since this will cause problems in the flow execution.
+-   User can run the full RTL-to-GDS flow and learn specific flow
+    sections independently. This allows users to learn the flow and tool
+    capabilities at their own pace, time and preference.
+-   Results shown, such as images or outputs of reports and logs, could
+    vary based on release updates. However, the main flow and command
+    structure should generally apply.
+
+Note: Please submit any problem found under Issues in the GitHub repository
+[here](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/issues).
+
+## Getting Started
+
+This section describes the environment setup to build ORFS and get ready
+to execute the RTL-to-GDS flow of the open-source design `ibex` using the
+`sky130hd` technology.
+
+`ibex` is a 32 bit RISC-V CPU core (`RV32IMC/EMC`) with a two-stage
+pipeline.
+
+### Setting Up The Environment
+
+Use the `bash` shell to run commands and scripts.
+
+Install OpenROAD Flow Scripts refer to the [Getting Started with
+OpenROAD Flow](../user/GettingStarted.md) documentation.
+
+If ORFS is already installed but needs updating, then use the
+following commands:
+
+```
+cd OpenROAD-flow-scripts
+git checkout master
+git pull
+git submodule update
```

Comment:
Why not just `./build_openroad.sh --latest`. This does the same thing as well as also building and installing.

Reply:
Actually please disregard this. This only updates the tools, not the ORFS repository. Please keep as is.